core/tests/org.openjdk.jmc.flightrecorder.rules.jdk.test/baseline/Generated_JfrRuleBaseline.xml

```diff
@@ -0,0 +1,4997 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
What's the purpose of this and the next file?

Reply:
This one was accidentally checked in. I'll remove it and add it to the .gitignore. Good catch.